### PR TITLE
Do not test empty OVAL 5.10 definition rendered by Jinja

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Skip testing empty rendered OVAL definition.
- Improve error message to indicate which OVAL definition failed `validate-parse-affected` test.

#### Rationale:

- When building only OVAL 5.10 content only, `validate-parse-affected` test fails because it tries to test a definition that was rendered empty by Jinja.
- Some definitions can be empty when building OVAL 5.10 only content.
  - For example: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_setting/oval/shared.xml
- Fixes Jenkins job [scap-security-guide-nightly-oval510-zip](https://jenkins.complianceascode.io/view/Maintenance%20Jobs/job/scap-security-guide-nightly-oval510-zip/)

### CODE REVIEW

File: tests/test_parse_affected.py [hunk 1]
```diff
@@ -54,13 +54,24 @@ def main():
 
             for oval in ssg.rules.get_rule_dir_ovals(rule_dir):
                 xml_content = ssg.jinja.process_file_with_macros(oval, env_yaml)
+                # Some OVAL definitions may render to and an empty definition
```
Comment (ggbecker) [inline tests/test_parse_affected.py]: remove `and`.

Comment (yuumasato) [reply tests/test_parse_affected.py]: Fixed by force.

Comment (ggbecker) [reply tests/test_parse_affected.py]: Thanks.

**Comment by yuumasato** [comment]
Tests failed in Fedora for random reasons.
@openscap-ci test this please

**Comment by ggbecker** [comment]
Thanks for the patch.

**Comment by yuumasato** [comment]
Thanks for review

**Pull request merged**